.github/workflows/python.yml

```diff
@@ -0,0 +1,36 @@
+name: Python
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
```

Comment:
can you remove the branches limitation? This also applies to a fork so if you make some changes and push it to a branch of your own fork, the tests will not run.

Reply:
Sure!